Formula/livechart

```diff
@@ -0,0 +1,27 @@
+class Livechart < Formula
+  desc "Real-time charting library for Vala and GTK3 based on Cairo"
+  homepage "https://github.com/lcallarec/live-chart"
+  url "https://github.com/lcallarec/live-chart/archive/refs/tags/1.9.1.tar.gz"
+
+  sha256 "860d57bd633bd1c8c6d387190d8d5ace39fe4a8abb26c0a40bb0c8d263c7a3c0"
+  license "MIT"
+
+  depends_on "gettext" => :build
+  depends_on "gobject-introspection" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => [:build, :test]
+  depends_on "vala" => :build
+  depends_on "gtk4"
+
+  def install
+    system "meson", "setup", "build", *std_meson_args
+    system "meson", "compile", "-C", "build", "--verbose"
+    system "meson", "install", "-C", "build"
+  end
+
+  test do

```

Comment:
This should test the actual library, not just the pc files

Reply:
Yes, now yet have a simple solution to test, so just put the pc files check code in it.

Reply:
That isn't a good enough test.